tests/bsim/bluetooth/host/misc/hfc/src/main.c

```diff
@@ -0,0 +1,444 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/hci.h>
+#include <zephyr/bluetooth/l2cap.h>
+#include <zephyr/bluetooth/att.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/bluetooth/uuid.h>
+#include <zephyr/bluetooth/conn.h>
+
+#include "testlib/att_read.h"
+
+#include <argparse.h>		/* For get_device_nbr() */
+#include "utils.h"
+#include "bstests.h"
+
+#define strucc struct
+#define staticc static
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(main, LOG_LEVEL_INF);
+
+DEFINE_FLAG(is_connected);
+DEFINE_FLAG(is_subscribed);
+
+static struct bt_conn *dconn;
+
+#define NUM_NOTIFICATIONS 200
+#define P1_START_DELAY K_SECONDS(1)
```

Comment:
What is `P1` here?
Peripheral 1?
Phase 1?

Reply:
it was peripheral 1, yes. I just forgot to update it.